### ISSUE TITLE
Can `(when)` option be applied to Spine Time types?

### ISSUE THREAD

**Issue by alexander-yevsyukov** [SpineEventEngine/base#316]
Now `options.proto` says that the option applies only to `Timestamp` fields. 
Can we use it to Spine Type fields too?

**Comment by dmdashenkov** [comment]
This is a duplicate of #63.

**Comment by alexander-yevsyukov** [comment]
So, the answer is: “Not at the moment, but it would be able after #63”. I'm closing this issue.
